### PR TITLE
Harpy Language!

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR adds a new Language specific to the Harpy species, which is composed of syllable charts taken from Polish, Serbian, Greek, and Turkish. It uses Cambria as its font, which is modestly different from the standard font, but still reasonably readable. This also adds a Language menu trait that allows people to buy the language with a trait point.

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

# Changelog

:cl:
- add: Harpies now have their own species-specific language, called Valyrian Standard
- add: Valyrian Standard can now be bought as an extra language in the Traits menu.

### CODE REVIEW

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Ofc its a language trait also.

Anyway you forgot the Translators.
Infact i think some languages are translator forgotten, at least add the harpy language translator to the rest.

**Comment by FoxxoTrystan** [comment]
And i love, only .yml changes to add languages

File: Resources/Locale/en-US/traits/traits.ftl [hunk 1]
```diff
@@ -314,3 +314,9 @@ trait-description-Spinarette =
 trait-name-AddictionNicotine = Nicotine Addiction
 trait-description-AddictionNicotine =
     You have an addiction to Nicotine, and will require frequent smoke breaks to keep your mood in check.
+
+trait-name-ValyrianStandard = Valyrian Standard
+trait-description-ValyrianStandard =
+    A language descended from eastern european languages of old earth - Valyrian Standard is the commonly spoken tongue of Harpies brought up on their homeworld of Valyrian 4b
+    It is rarely spoken outside of the worlds of its native speakers, and has in modern times been supplanted by the 'Conlangs of the Sol Alliance.
+    Its speakers are those who wish to uphold the traditions and beliefs of ancient peoples from before the colonial era.
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/traits/traits.ftl:322]:
```suggestion
    Its speakers are those who wish to uphold the traditions and beliefs of ancient peoples from before the colonial era.

```

Comment (DEATHB4DEFEAT) [reply Resources/Locale/en-US/traits/traits.ftl:322]: How do you keep doing this?

File: Resources/Prototypes/Language/Species-Specific/harpy.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    - ou
+    - bl
+    - dr
+    - fl
+    - gr
+    - pr
+    - sz
+    - cz
+    - zk
+    - jn
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Language/Species-Specific/harpy.yml:116]:
```suggestion
    - jn

```

File: Resources/Prototypes/Traits/languages.yml [hunk 3]
```diff
@@ -64,4 +64,16 @@
...
+  id: ValyrianStandard
+  category: TraitsSpeechLanguages
+  points: 1
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: TraitsLanguagesBasic
+  languagesSpoken:
+    - ValyrianStandard
+  languagesUnderstood:
+    - ValyrianStandard
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/languages.yml:79]:
```suggestion
    - ValyrianStandard

```

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Locale/en-US/traits/traits.ftl (hunk 1 above)
Comment (VMSolidus) [reply Resources/Locale/en-US/traits/traits.ftl:322]: I have no idea? When I write the PR in VSCode, it includes the Newline.

Comment (DEATHB4DEFEAT) [reply Resources/Locale/en-US/traits/traits.ftl]: You have EditorConfig right?

**Comment by VMSolidus** [comment]
Translators have been added for Valyrian Standard

**Review by FoxxoTrystan — APPROVED** [review]

**Pull request merged**